tools/apiview/parsers/csharp-api-parser/CSharpAPIParser/TreeToken/CodeFileBuilder.cs

```diff
@@ -517,8 +517,10 @@ private void BuildAttributes(List<ReviewLine> reviewLines, ImmutableArray<Attrib
                             attributeLine.AddToken(ReviewToken.CreatePunctuationToken(SyntaxKind.EqualsToken));
                             BuildTypedConstant(attributeLine, argument.Value);
                         }
+                        attributeLine.Tokens.Last().HasSuffixSpace = false;
```

Comment:
@christothes are there any other cases where there needs to be special considerations for attributes?